packages/neuron-ui/src/contexts/transfer.ts

```diff
@@ -0,0 +1,12 @@
+import { createContext } from 'react'
+
+interface ITransfer {
```

Comment:
Do not prefix `I` for interfaces.

Reply:
ok, I referenced settings.ts and I will rename it.